test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -1205,6 +1205,19 @@ Describe "Extended FileSystem Item/Content Cmdlet Provider Tests" -Tags "Feature
             $result.Name | Should -BeExactly $testDir
         }
 
+        It "Verify Directory creation when path relative to current PSDrive is empty" {
+            try {
+                New-Item "NewDirectory" -ItemType Directory -Force
+                $newPSDrive = New-PSDrive -Name "NewPSDrive" -PSProvider FileSystem -Root "NewDirectory"
```

Comment:
Here we create PSDrive with relative root path - this leads to unpredictable results (the psdrive base path is taken as OS cwd!). Please use absolute path like:
```suggestion
                $rootDir = New-Item "NewDirectory" -ItemType Directory -Force
                $rootPath = $rootDir.FullName
                $newPSDrive = New-PSDrive -Name "NewPSDrive" -PSProvider FileSystem -Root $rootPath
```